src/testdir/test_crypt.vim

```diff
@@ -438,4 +438,27 @@ func Test_crypt_set_key_segfault()
   bwipe!
 endfunc
 
+func Test_crypt_set_key_disallow_append_subtract()
```

Comment:
This should also need `CheckFeature cryptv`

Reply:
That already exists at the top of the file. Every test in this file is to test the crypto feature so I don't think we need to add that in each test?

The only other feature checking the tests do are `CheckFeature sodium` since that's an additional requirement.

Reply:
Oops, didn't notice the name of the file.